assets/js/components/auth/MagicAuthenticate.jsx

```diff
@@ -1,39 +1,127 @@
-import React, { useState } from 'react';
+import React, { useState, useEffect } from 'react';
 import { useSelector } from "react-redux";
 import { useHistory } from "react-router-dom";
 import { Button, Card, Typography, Input, Form } from 'antd';
 import GoogleOutlined from "@ant-design/icons/GoogleOutlined";
 import { loginUser, loginGoogleUser } from '../../actions/magic';
 import AuthLayout from '../common/AuthLayout'
 import Logo from '../../../img/symbol.svg'
+import * as rest from '../../util/rest';
+import { displayError } from '../../util/messages';
 const { Text, Title } = Typography
 
+const styles = {
+  none: {
+    opacity: 0,
+    position: 'absolute',
+    top: 0,
+    left: 0,
+    height: 0,
+    width: 0,
+    zIndex: -1,
+  }
+}
+
 const MagicAuthenticate = () => {
   const history = useHistory()
   const [email, setEmail] = useState('');
   const [loading, setLoading] = useState('');
-  const [error, setError] = useState(null);
+  const [name, setName] = useState('')
+  const recaptchaSiteKey = process.env.RECAPTCHA_SITE_KEY || '6Len2logAAAAALaqL5cECU0Vl7JJqqbIQX6IgWz6'
+  const useRecaptchaForAuth = process.env.USE_RECAPTCHA_FOR_AUTH === 'true' // set to true here to test recaptcha in dev
```

Comment:
don't need this comment anymore, right? we can keep it like this and just not set the key in environments where we don't want captcha

Reply:
you need it for dev since you cant easily set process envs in dev

Reply:
this var isn't being surface up through the backend

Reply:
oh i thought you'd just check for `RECAPTCHA_SITE_KEY` env var